### ISSUE TITLE
Set tolerances of the nonlinear solver for individual unknowns

### ISSUE THREAD

**Issue by Embreus** [chalmersplasmatheory/DREAM#37]
Implement interface support to adjust the absolute and relative tolerances of the non-linear solver of single unknowns. 

The present method solver.SetTolerance(reltol) could be a shortcut to set tolerances of all unknowns, but in the future it would be good to set tolerances of individual unknowns somehow.

**Comment by Embreus** [comment]
Not sure how the interface should look though without being too clumsy?
solver.SetTolerance(reltol_n_re, abstol_n_re, reltol_T_cold, abstol_T_col, ...) 
seems really cumbersome and generalises poorly with new unknowns, but the functionality would be useful.

Let each unknown in the eqsys have its own SetTolerance method?

**Comment by hoppe93** [comment]
Since I think we should be able to set tolerances separately for the ``Solver`` and ``TimeStepper``, I would probably favour an interface similar to the new ``TransportSettings`` interface, i.e. we have a ``ToleranceSettings`` class which is created for ``Solver`` and ``TimeStepper``. The ``ToleranceSettings`` keeps a list of unknowns and their relative and absolute tolerances. By default the list is empty, in which case the kernel applies default tolerances?

Next, to set a tolerance, I imagine we would have something like
```python
ds.solver.tolerances.set('n_re', abstol=1e5, reltol=1e-6)
```
where we specify the name of the unknown as a string. To make things a bit safer, we could add a list of unknown names as constants to (perhaps) ``EquationSystem`` which can be accessed and checked against by the ``ToleranceSettings`` object (to avoid misspellings).

We could then also imagine setting all tolerances in one go with a syntax like
```python
ds.solver.tolerances.set(abstol=1e5, reltol=1e-6)
```

**Comment by Embreus** [comment]
Fixed by hoppe93 in 082969c
